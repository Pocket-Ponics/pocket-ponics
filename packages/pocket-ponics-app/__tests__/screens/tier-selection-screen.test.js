import React from 'react'
import { shallow } from 'enzyme'

import TierSelectionScreen from '../../screens/tier-selection-screen'

describe('Tier Selection Screen', () => {
	test('renders correctly', () => {
		global.plants = [{},{},{},{},{}]
		const tree = shallow(
			<TierSelectionScreen />
		).html()
		expect(tree).toMatchSnapshot()
	})
})