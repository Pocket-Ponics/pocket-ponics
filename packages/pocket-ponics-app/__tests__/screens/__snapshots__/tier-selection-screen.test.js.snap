// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tier Selection Screen renders correctly 1`] = `"<View style=\\"flex:1\\"><View style=\\"background-color:#472600;color:#FFFFFF;padding:20px;flex-direction:column;flex:1\\"><Modal animationType=\\"slide\\"><View style=\\"justify-content:flex-end;flex:1\\"><View style=\\"height:400px;background-color:#FFFFFF;border-radius:10px\\"><Text style=\\"font-size:32px;margin-bottom:20px;color:#000000;align-self:center;margin-top:20px\\">Plant Choices</Text><RCTScrollView data=\\"[object Object]\\" numColumns=\\"3\\" initialNumToRender=\\"10\\" maxToRenderPerBatch=\\"10\\" scrollEventThrottle=\\"50\\" updateCellsBatchingPeriod=\\"50\\" windowSize=\\"21\\" viewabilityConfigCallbackPairs=\\"\\" stickyHeaderIndices=\\"\\"><View><View><View style=\\"flex-direction:row\\"><View style=\\"flex:1;flex-direction:column;margin:5px;justify-content:center;align-items:center;opacity:1\\"><Image source=\\"1\\" style=\\"resize-mode:contain;width:100px;height:100px;border-radius:50px;background-color:#472600\\"></Image><Text>Tomato</Text></View></View></View></View></RCTScrollView></View></View></Modal><Text style=\\"color:#FFFFFF;font-size:32px;margin-bottom:20px;text-align:center\\">Select plants</Text><Text style=\\"color:#FFFFFF;font-size:18px;margin-bottom:20px\\">Name the greenhouse, and then tap each tier of the greenhouse to assign plants</Text><View style=\\"flex-direction:row;justify-content:center;align-items:center\\"><Text style=\\"color:#FFFFFF;font-size:18px;line-height:45;padding-right:10px;font-weight:bold\\">Name:</Text><TextInput underlineColorAndroid=\\"transparent\\" style=\\"flex:1;height:45px;border-radius:45px;font-size:18px;padding-left:20px;background-color:rgba(0, 0, 0, 0.35);color:#FFFFFF\\" placeholder=\\"Greenhouse Name\\" placeholderTextColor=\\"rgba(255, 255, 255, 0.7)\\" value=\\"\\" autoCapitalize=\\"words\\"></TextInput></View><View style=\\"flex:1\\"><View style=\\"color:#472600;align-items:center;justify-content:center;margin:10px;flex:1\\"><View style=\\"flex:1.8;width:100%;align-items:center;justify-content:center;opacity:1\\"><View style=\\"flex:1;flex-direction:row;justify-content:center;aspect-ratio:1.1904761904761905px\\"><Image source=\\"1\\" style=\\"0:[object Object];1:[object Object];2:[object Object]\\"></Image></View></View><View style=\\"flex:1;width:100%;align-items:center;justify-content:center;opacity:1\\"><View style=\\"flex:1;flex-direction:row;justify-content:center;aspect-ratio:2.142857142857143px\\"><Image source=\\"1\\" style=\\"0:[object Object];1:[object Object];2:[object Object]\\"></Image></View></View><View style=\\"flex:1;width:100%;align-items:center;justify-content:center;opacity:1\\"><View style=\\"flex:1;flex-direction:row;justify-content:center;aspect-ratio:2.142857142857143px\\"><Image source=\\"1\\" style=\\"0:[object Object];1:[object Object];2:[object Object]\\"></Image></View></View><View style=\\"flex:1;width:100%;align-items:center;justify-content:center;opacity:1\\"><View style=\\"flex:1;flex-direction:row;justify-content:center;aspect-ratio:2.142857142857143px\\"><Image source=\\"1\\" style=\\"0:[object Object];1:[object Object];2:[object Object]\\"></Image></View></View></View></View><View style=\\"background-color:#456E31;border-radius:5px;padding:10px;margin:10px;align-items:center;opacity:0.3\\"><Text style=\\"color:#FFFFFF;font-size:22px\\">Continue Setup</Text></View><View style=\\"align-items:center;opacity:1\\"><Text style=\\"color:#84AB71;font-size:22px\\">Cancel Setup</Text></View></View></View>"`;
